modules/smtp/modules.php

```diff
@@ -407,6 +408,7 @@ public function process() {
             $this->out('compose_draft', $draft);
         }
         $this->out('smtp_compose_type', $compose_type);
+        $this->out('user_settings', $settings);
```

Comment:
Instead of passing the whole user_settings you can just pass enable_compose_delivery_receipt_setting

Reply:
Working on it now

Reply:
@josaphatim done.